layout/_layout.swig

```diff
@@ -115,9 +115,28 @@
   {% include '_third-party/needsharebutton.swig' %}
   {% include '_third-party/rating.swig' %}
   {% include '_third-party/pangu.swig' %}
-  {% include '_third-party/scroll-cookie.swig' %}
-  {% include '_third-party/exturl.swig' %}
   {% include '_third-party/bookmark.swig' %}
   {% include '_third-party/copy-code.swig' %}
+
+  {% if theme.exturl %}
```

Comment:
Don't do this.

Reply:
They are obviously different from other `third-party` libraries - their dependencies are located in `/source/js/src`. Put these files to `_scripts` would be better.
![2019-03-04 11 23 20](https://user-images.githubusercontent.com/16272760/53742954-be683500-3ed4-11e9-8e2b-9f4af8ecc3e6.png)
What's your opinion?

Reply:
Don't join New Features and possible Optimizations at once. Please, fix it.

Reply:
Fixed